BioForm/api/views.py

```diff
@@ -0,0 +1,143 @@
+import datetime
+import jwt
+from profiles.models import *
+from rest_framework import status, viewsets
+from rest_framework.response import Response
+from rest_framework.exceptions import AuthenticationFailed
+from rest_framework.views import APIView
+from .serializers import *
+from .permissions import IsJWTAuthenticated, IsUsersObject
+
+def serialize_user(user):
```

Comment:
I can't find any usage of it, is it required?

Reply:
serialize_user function was being used prior to class views. It was responsible for serializing user when the view that displayed current user signed in was called. I made that view to see if I was logged in or not and after adding custom permission message, that feature was not needed anymore so I removed the view. 

I have removed this function as well now.